core/templates/dev/head/pages/exploration_player/correctness_footer_directive.html

```diff
@@ -0,0 +1,34 @@
+<style>
+  correctness-footer .inner-container {
+    background: white;
```

Comment:
Alphabetize. That will also show you that you have e.g. both background and background-color...

Reply:
Sorry , I didn't get it. Can you give an example?

Reply:
background, then background-color, then border, etc.

And your background/background-color rules are in conflict. You just need one or the other. There may be other conflicts as well. Remove redundant stuff you don't need.

Reply:
Done!